nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -398,15 +392,35 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
 
             types.forEach((fileType) => {
                 const appId = this.typesToFiles[fileType].appId;
+                const paramsToRemap = JSON.parse(JSON.stringify(params[fileType].params));
```

Comment:
this ensures that the data getting sent off to the xsv template generator uses the appropriate value -- e.g. the form display value or the value in the dynamic dropdown for the species name